### PR TITLE
fix(insured-bridge): remove fixedpoint from InsuredBridge math to save gas

### PR DESCRIPTION
**Pull request by chrismaree**
**Motivation**

Previously, the insured-bridge used `FixedPoint` for computations. Removing this from the math makes it a bit harder to read but saves a non negligible amount of gas. 

**Summary**

Re-writes numeric operations to work without `FixedPoint`. Note we still need to include the fixed point library to extract the `finalFee` from the call to the `Store` via `_getStore().computeFinalFee(address(l1Token)).rawValue`

**Gas Difference:**
`relayDeposit` before PR:542868. after PR: 539943. saving: 2925
`speedUpRelay` before PR: 94842. after PR: 93958. saving: 884
`settleRelay` before PR: 309697. after PR: 306287. saving: 3410
total saving: 7219 

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [X]  Existing tests adequate, no new tests required
- [X]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

**Review by nicholaspai — APPROVED** [review]
LGTM this was much needed!

File: packages/core/contracts/insured-bridge/BridgePool.sol [hunk 1]
```diff
@@ -144,8 +144,7 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20, MultiCaller
     function addLiquidity(uint256 l1TokenAmount) public {
         l1Token.safeTransferFrom(msg.sender, address(this), l1TokenAmount);
 
-        uint256 lpTokensToMint =
-            FixedPoint.Unsigned(l1TokenAmount).div(FixedPoint.Unsigned(exchangeRateCurrent())).rawValue;
+        uint256 lpTokensToMint = (l1TokenAmount * 1e18) / exchangeRateCurrent();
```
Comment (nicholaspai) [inline packages/core/contracts/insured-bridge/BridgePool.sol:147]: This is all possible because solidity 8 uses safe math by default right?

Comment (chrismaree) [reply packages/core/contracts/insured-bridge/BridgePool.sol:147]: yup :)

**Comment by chrismaree** [comment]
@mrice32 I'd just like to double-check you are OK with how we are doing math in this PR.

**Review by mrice32 — APPROVED** [review]
LGTM!

**Pull request merged**